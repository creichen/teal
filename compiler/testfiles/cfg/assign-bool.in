fun f(x) = {
  x := x or 1;
}
