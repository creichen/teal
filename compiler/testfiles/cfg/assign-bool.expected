Entry#0 @ 1 :
  -> Access#1
VarDecl#0 @ 1 :
Access#0 @ 2 :
  -> AssignStmt#0
AssignStmt#0 @ 2 :
  -> Exit#0
Access#1 @ 2 :
  -> IntConstant#0
OrExpr#0 @ 2 :
  -> Access#0
IntConstant#0 @ 2 :
  -> OrExpr#0
Exit#0 @ 3 :
